ethereum/ts-scripts/relayer/config/testnet/chains.json

```diff
@@ -36,43 +36,14 @@
       "chainId": 16,
       "rpc": "https://rpc.testnet.moonbeam.network",
       "wormholeAddress": "0xa5B7D85a8f27dd7907dc8FdC21FA5657D5E2F901"
-    }
-  ],
-  "registrationChains": [
-    {
-      "description": "Avalanche testnet fuji",
-      "evmNetworkId": 43113,
-      "chainId": 6,
-      "rpc": "https://api.avax-test.network/ext/bc/C/rpc",
-      "wormholeAddress": "0x7bbcE28e64B3F8b84d876Ab298393c38ad7aac4C"
-    },
-    {
-      "description": "Celo Testnet",
-      "evmNetworkId": 44787,
-      "chainId": 14,
-      "rpc": "https://alfajores-forno.celo-testnet.org",
-      "wormholeAddress": "0x88505117CA88e7dd2eC6EA1E13f0948db2D50D56"
-    },
-    {
-      "description": "BSC Testnet",
-      "evmNetworkId": 97,
-      "chainId": 4,
-      "rpc": "https://bsc-testnet.public.blastapi.io",
-      "wormholeAddress": "0x68605AD7b15c732a30b1BbC62BE8F2A509D74b4D"
     },
     {
-      "description": "Mumbai",
-      "evmNetworkId": 80001,
-      "chainId": 5,
-      "rpc": "https://matic-mumbai.chainstacklabs.com",
-      "wormholeAddress": "0x0CBE91CF822c73C2315FB05100C2F714765d5c20"
-    },
-    {
-      "description": "Moonbase Alpha",
-      "evmNetworkId": 1287,
-      "chainId": 16,
-      "rpc": "https://rpc.testnet.moonbeam.network",
-      "wormholeAddress": "0xa5B7D85a8f27dd7907dc8FdC21FA5657D5E2F901"
+      "description": "Base Testnet",
+      "evmNetworkId": 84531,
+      "chainId": 30,
+      "rpc": "https://goerli.base.org",
+      "wormholeAddress": "0x23908A62110e21C04F3A4e011d24F901F911744A"
     }
-  ]
+  ],
+  "operatingChains": [30]
```

Comment:
just curious... is this the number of items in the above array? (I didn't check if the number is right :stuck_out_tongue_closed_eyes:

Reply:
haha no it just indicates that I was performing actions for 'base testnet'

Reply:
I'll remove this from the PR perhaps